src/Sylius/Bundle/AttributeBundle/Resources/views/Types/datetime.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if attribute.attribute.configuration['format'] is defined %}
+    <td>{{ attribute.value|date(attribute.attribute.configuration['format']) }}</td>
+{% else %}
+    <td>{{ attribute.value|date('d/m/Y H:i:s') }}</td>
```

Comment:
I don't think we should hardcode this here, just call ``|date`` and that's it. It will use default format.